projects/fsastorefrontlib/src/cms-components/comparison-table/comparison-table-container/comparison-table-container.component.ts

```diff
@@ -81,6 +82,28 @@ export class ComparisonTableContainerComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Sorts and returns available tabs by title
+   *
+   * @returns CMSComparisonTabComponent[] observable
+   */
+  private getAvailableTabsSortedByTitle(): Observable<
+    CMSComparisonTabComponent[]
+  > {
+    return this.comparisonTableService.availableTab$.pipe(
+      map(tab => this.sortByTitle(tab))
+    );
+  }
+
+  /**
+   * Sorts tabs by title
+   *
+   * @returns CMSComparisonTabComponent[]
+   */
+  private sortByTitle(tab: CMSComparisonTabComponent[]) {
```

Comment:
Perhaps this level of separation of concerns is not needed. This method is very specific and not reusable, so I would suggest we remove it and move the logic to `getAvailableTabsSortedByTitle` method.